examples/Model.elm

```diff
@@ -364,7 +392,7 @@ fetchRecordList model =
             |> Kinto.getList recordResource
             |> Kinto.sortBy [ sortColumn ]
             |> limiter
-            |> Kinto.send FetchRecordsResponse
+            |> Kinto.send (FetchRecordsResponse False)
```

Comment:
This is now unneeded after https://github.com/Kinto/elm-kinto/pull/27/commits/69eb88851090c63c3dc98f6ea1151579f2468c6c